src/_tests_/Home.test.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+
+import '@testing-library/jest-dom';
+// Используем свою функцию render, а не render
+// из @testing-library/react (our custom utils
+// also re-export everything from RTL
+// so we can import fireEvent and screen here
+// as well)
+import { render, screen, cleanup } from './test-utils';
+
+import App from '../App';
+import mockCards from '../../__mocks__/mockCards';
+
+afterEach(() => {
+  cleanup();
+});
+
+describe('Home page testing when the results are received from the newsapi', () => {
```

Comment:
when you add a colon at the end of describe description string, it becomes a bit readable
![image](https://user-images.githubusercontent.com/58717777/132826461-5dc3ca93-ed2f-443b-8803-b1f45f540afa.png)